software_development/INTELLIJ.md

```diff
@@ -68,3 +68,49 @@ It is especially good at working with large multi-module projects such as GeoNet
    * Use *Build Project* to compile, allowing methods to be rewritten interactively.
    * Use *Maven* tool window to run `mvn process-resources` to
      copy changes into the running web application.
+
+## IntelliJ Settings / Preferences
+
+1. Open settings:
+   
+   * **Window > Preferences** (Windows and Linux)
+   * **IntelliJ IDEA > Preferences** (macOS)
+
+2. Navigate to **Editor > Code Style > Java**.
+   
+   * Locate **Scheme** at the top of the page, and the actions menu.
+     
+   * Select **Import Scheme | Eclipse XML Profile **
+   
+   * Import the `code_quality/formatter.xml`
+
+3. Navigate to **Editor > File and Code Templates**.
+   
+   * Change to **Includes** tab
+     
+   * Define **File Header**:
+   
+     ```
+     //=============================================================================
+     //===	Copyright (C) 2001-2007 Food and Agriculture Organization of the
```

Comment:
What should the year be in this case? I see a lot of different years in the license within the code? Shouldn't they all be the same?

Reply:
They should be when the file was created, or if you are copying from another file when the original was created ...

Reply:
aside: I updated the template to fill in the current year